DigitalOffice/Tests/FileServiceTests/Mappers/FileMapperTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using FileService.Database.Entities;
+using FileService.Mappers;
+using FileService.Mappers.Interfaces;
+using FileService.Models;
+using NUnit.Framework;
+using System;
+
+namespace FileServiceUnitTests.Mappers
+{
+    public class FileMapperTests
+    {
+        private DbFile dbFile;
+        private FileRequest fileRequest;
+        private IMapper<DbFile, FileRequest> dbToRequestMapper;
+        private IMapper<FileRequest, DbFile> requestToDbMapper;
+
+        [SetUp]
+        public void Initialization()
+        {
+            dbToRequestMapper = new FileMapper();
+            requestToDbMapper = new FileMapper();
+
+            fileRequest = new FileRequest
+            {
+                Content = "RGlnaXRhbCBPZmA5Y2U=",
+                Extension = ".txt",
+                Name = "DigitalOfficeTestFile"
+            };
+
+            dbFile = new DbFile
+            {
+                Id = Guid.NewGuid(),
+                Content = Convert.FromBase64String("RGlnaXRhbCBPZmA5Y2U="),
+                Extension = ".txt",
+                IsActive = true,
+                Name = "DigitalOfficeTestFile"
+            };
+        }
+
+        [Test]
+        public void FailedMappingObjectIsNullTest()
+        {
+            fileRequest = null;
+            dbFile = null;
+
+            Assert.Throws<NullReferenceException>(() => dbToRequestMapper.Map(dbFile));
+            Assert.Throws<NullReferenceException>(() => requestToDbMapper.Map(fileRequest));
+        }
+
+        [Test]
+        public void SuccessfulRequestMappingTest()
+        {
+            var newFile = requestToDbMapper.Map(fileRequest);
+
+            var expectedFile = new DbFile
+            {
+                Id = fileRequest.Id,
+                Content = Convert.FromBase64String(fileRequest.Content),
+                Extension = fileRequest.Extension,
+                Name = fileRequest.Name,
+                IsActive = true,
+            };
+
+            Assert.IsInstanceOf<DateTime>(newFile.AddedOn);
+            Assert.IsInstanceOf<Guid>(newFile.Id);
+            Assert.AreEqual(expectedFile.Id, newFile.Id);
+            Assert.AreEqual(expectedFile.Content, newFile.Content);
+            Assert.AreEqual(expectedFile.Extension, newFile.Extension);
+            Assert.AreEqual(expectedFile.IsActive, newFile.IsActive);
```

Comment:
It is checks can be place in a separate method of class SerializerAssert ).

Reply:
> It is checks can be place in a separate method of class SerializerAssert ).

I specifically mentioned it in the commit description.
I cannot test it via serializerassert because of the AddedOn property.

Reply:
You can this only for AreEqual.

Reply:
So you suggest I add a separate method for these checks only? In a static class? I don't think it's a good idea.

I don't mind doing it though, but I disagree nonetheless.

Reply:
Fixed it in SerializerAssert.cs